datastore/additional_data/management/commands/rewrite_additional_data.py

```diff
@@ -17,6 +25,13 @@ def add_arguments(self, parser):
             help="The datagetter run id or latest",
         )
 
+        parser.add_argument(
+            "--grants-chunk-size",
+            type=int,
+            action="store",
+            help="Number of Grants to update simultaneously",
```

Comment:
Could do `default=DEFAULT_CHUNK_SIZE` to avoid need for logic later on?

Reply:
Done